### ISSUE TITLE
Licensing

### ISSUE THREAD

**Issue by virtuallynathan** [CAIDA/cc-common#4]
The license files for some of the sub-repos list one license, where the code lists another. 

Lots of stuff has GPLv2-like text, but mentions GPLv3 in the code. 

cc-common/libjsmn/Makefile.am:# the Free Software Foundation, either version 3 of the License, or
cc-common/libinterval3/rb_tree/Makefile.am:# the Free Software Foundation, either version 3 of the License, or
cc-common/libinterval3/Makefile.am:# the Free Software Foundation, either version 3 of the License, or
cc-common/libinterval3/interval_tree.h: * the Free Software Foundation, either version 3 of the License, or
cc-common/wandio_utils.c: * the Free Software Foundation, either version 3 of the License, or
cc-common/wandio_utils.h: * the Free Software Foundation, either version 3 of the License, or
cc-common/ip_utils.c: * the Free Software Foundation, either version 3 of the License, or
cc-common/Makefile.am:# the Free Software Foundation, either version 3 of the License, or
cc-common/jsmn_utils.c: * the Free Software Foundation, either version 3 of the License, or
cc-common/libcsv/Makefile.am:# the Free Software Foundation, either version 3 of the License, or
cc-common/libpatricia/Makefile.am:# the Free Software Foundation, either version 3 of the License, or
cc-common/akq.h: * the Free Software Foundation, either version 3 of the License, or
cc-common/jsmn_utils.h: * the Free Software Foundation, either version 3 of the License, or
cc-common/utils.c: * the Free Software Foundation, either version 3 of the License, or
cc-common/utils.h: * the Free Software Foundation, either version 3 of the License, or
cc-common/akarr.h: * the Free Software Foundation, either version 3 of the License, or
cc-common/ip_utils.h: * the Free Software Foundation, either version 3 of the License, or

Is all of this code meant to be GPLv3? 

Thanks,
Nathan

**Comment by alistairking** [comment]
Good catch. I think the files that mention GPLv3 are files that I created myself (e.g. a makefile in a sub-repo), whereas the other files are from the original project and thus have whatever license that project had.
I should probably put some effort into tidying all of this up (and making sure we don't have conflicting licenses..).
